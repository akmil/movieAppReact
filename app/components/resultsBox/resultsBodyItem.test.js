import React from 'react';
// import renderer from 'react-test-renderer';
import {ResultsBodyItem } from "./ResultsBodyItem";
import { shallow } from 'enzyme'

describe('ResultsBody', function () {
    it('should render without errors ResultsBody', () => {
        const wrapper = shallow(<ResultsBodyItem  title={'title'} genres={['genres']} release_date={'foo'} poster_path ={'foo'}/>);
        expect(wrapper).toMatchSnapshot();
    });
});