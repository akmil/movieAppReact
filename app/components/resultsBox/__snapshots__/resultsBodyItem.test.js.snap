// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResultsBody should render without errors ResultsBody 1`] = `
<li>
  <img
    alt="title"
    src="foo"
  />
  <div
    className="col-50"
  >
    <p
      className="title"
    >
      title
    </p>
    <p>
      <span
        className="genre"
        key="0"
      >
        genres
         
      </span>
    </p>
  </div>
  <div
    className="col-50 text-align-right"
  >
    <span
      className="release-date"
    >
      foo
    </span>
  </div>
</li>
`;
